shortfin/python/shortfin_apps/llm/components/batcher.py

```diff
@@ -375,6 +357,14 @@ async def run(self):
 class PrefillExecutorProcess(LlmExecutorProcess):
     """Executes a prefill batch."""
 
+    tokens_host_cache: dict[tuple[int, int], sfnp.device_array] = {}
```

Comment:
These would be globally shared between all executor processes with no cleanup. If we stopped the service then restarted it, then it would have invalid contents. We should not have static global state carried between processes as this would cause very bad outcomes in multi-instance.